cmd/all-in-one/Dockerfile.aarch64

```diff
@@ -0,0 +1,34 @@
+FROM alpine:latest as certs
+RUN apk add --update --no-cache ca-certificates
+
+FROM scratch
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+# Agent zipkin.thrift compact
+EXPOSE 5775/udp
+
+# Agent jaeger.thrift compact
+EXPOSE 6831/udp
+
+# Agent jaeger.thrift binary
+EXPOSE 6832/udp
+
+# Agent config HTTP
+EXPOSE 5778
+
+# Collector HTTP
+EXPOSE 14268
+
+# Collector gRPC
+EXPOSE 14250
+
+# Web HTTP
+EXPOSE 16686
+
+COPY all-in-one-linux-arm64 /go/bin/
```

Comment:
couldn't we instead templatize the main Dockerfile?

Reply:
Yes of course we can, however I think we probably need to add some workaround code such as `RUN mv all-in-one-linux* all-in-one-linux` before copying to unify the binary name for each architecture, which seems a little bit redundant, but if the community prefer a single Dockerfile, I can change the code of course :))

What do you think? Any suggestions?

Reply:
`all-in-one-linux*` won't necessarily help as it would match all binaries. We need to use some variable to specify which binary to copy (and I would copy it into a fixed name like `/go/bin/all-in-one`.

I certainly don't want maintaining a copy-pasta Dockerfile which is only different in one line.

Reply:
Hi @yurishkuro, sorry about the late reply and thanks again for the comments, makes really good sense. However I do have one question before updating:

For now, on x86_64 machines, all binaries will be only named as `<NAME>-linux` while on arm64 and s390x machines it will be named as `<NAME>-linux-<ARCH>`. Although it is implementable using docker build args, do you mind changing the x86_64 binaries to `<NAME>-linux-amd64` in order to unify the names? Will this affect the releasing policy or any other possible hidden problems?

Reply:
yeah, I was thinking the same. 

@jaegertracing/jaeger-maintainers any concerns with renaming the binaries consistently?

Reply:
WoW that is a really fast reply, thanks!

Anyway by any means I will try to updating this PR later today using the docker build args, it should not break anything but personally I think it is not the best...We can have further discussion in this PR about this naming issue.

Reply:
@yurishkuro PR updated. It works both on my x86_64 and arm64 machines, however if there are any better implementations or commens, please kindly propose them ;))